### ISSUE TITLE
Ease editor only allows zoom out to about 50% of the area

### ISSUE THREAD

**Issue by F3der1co** [Scirra/Construct-3-bugs#6103]
## Problem description
the ease editor only allows zooming out to see about 50% of the total area.

## Attach a .c3p
this is not project specific at all

## Steps to reproduce

1. open the ease editor
2. press the - magnification icon until you cannot zoom out further

## Observed result

it didn't zoom out all the way (you still can scroll up and down no matter what aspect ratio the window has)

## Expected result

it zoomed out all the way

## More details

additionally,  shift+scroll-wheel should allow zooming when the ease-editor is highlighted imo (or any of the other typical zooming inputs Construct 3 uses)

**Comment by DiegoScirra** [comment]
At the initial zoom level, the editor shows the full horizontal span of the curve. The vertical space is a little bit different because it is technically infinite, as there is no hard limit to the position the anchor points can take. You could keep zooming out, but there would always be more vertical space for the curve to take, as long as you can drag the anchor points further and further away, so there needs to be a cut off point.

The current limit at roughly 900% and -900% at the minimum zoom level is arbitrary. For practical purposes most people would keep a custom curve somewhere in the  100% and -100% range anyway, so it should be ok.

As for the missing zooming shortcuts, that will be added sometime in the future, but it is not pressing.

**Comment by F3der1co** [comment]
Yea, you are right! Thinking about it more the ease editor should allow vertical scolling even when zoomed out all the way.
The original reason I did this report was that when I experimented I couldn't zoom out to see the full ease curve. But to be fair I was at extreme values, so that is an edge case.
But I think ideally the lowest zoom level should always have the full curve in view. This would require dynamic zoom steps instead of fixed ones though.
But as this is such an edge case idk if it's even worth to fix.
